### PR TITLE
Upgrade packages

### PR DESCRIPTION
**Pull request by Harshith46**
update packages: 
- loadash: 4.17.15 to 4.17.19
- acorn: 5.7.3 to 5.7.4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quintype/backend",
-  "version": "1.3.0",
+  "version": "1.3.0-upgrade-package.1",
```
Comment (Deokumarjnu) [inline package.json:3]: remove this while merging the pr

**Review by Deokumarjnu — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Harshith46) [reply package.json:3]: it gets automatically updated

Comment (Deokumarjnu) [reply package.json:3]: hey, it looks like a beta version, can you verify this.

Comment (Harshith46) [reply package.json:3]: no can you take a look into standard-version-release file.

**Review by atigrepravin — APPROVED** [review]

**Pull request merged**